internsystem-backend/environments/prod/settings_local.py

```diff
@@ -0,0 +1,55 @@
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+if 'DJANGO_SECRET_KEY' not in os.environ:
+    raise Exception('Missing DJANGO_SECRET_KEY environment variable')
+SECRET_KEY = os.environ['DJANGO_SECRET_KEY']
+
+# see https://docs.djangoproject.com/en/1.7/ref/settings/#std:setting-DEBUG
+DEBUG = 'DJANGO_DEBUG' in os.environ
+
+# enable SAML if wanted
+# see setup_saml.sh
+ENABLE_SAML = 'DJANGO_ENABLE_SAML' in os.environ
+SAML_FOLDER = os.path.join(BASE_DIR, 'samlauth', 'prod')
+
+# see https://docs.djangoproject.com/en/1.7/ref/settings/#std:setting-ALLOWED_HOSTS
+#ALLOWED_HOSTS = [
+#    'something',
+#]
+
+# if collectstatic is used, set up this
+STATIC_ROOT = "/usr/src/static/"
+
+# example for postgres configuration
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': os.environ['POSTGRES_NAME'],
```

Comment:
With Docker it is normal to inject such environment variables when the container is run, see `run.sh`.